eos-factory-reset.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Endless factory reset service
+ConditionPathExists=/var/eos-factory-reset
+Conflicts=getty@tty1.service gdm.service plymouth-start.service
+After=systemd-user-sessions.service getty@tty1.service plymouth-quit.service gdm.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/lib/eos-factory-reset/eos-factory-reset-helper
+ExecStartPost=systemctl reboot
```

Comment:
I believe this step will not be run if the `ExecStart=` line exits unsuccessfully, so on error the system will just stay at the blank/splash screen forever until the user pulls the plug.